client/src/modules/users/users.routes.js

```diff
@@ -47,7 +47,15 @@ angular.module('bhima.routes')
         },
         onEnter :['$uibModal', depotManagementModal],
         onExit : ['$uibModalStack', closeModal]
-      })      
+      })
+      .state('users.cashBoxManagement', {
+        url : '/:id/cashBoxManagement',
```

Comment:
Since your server URL is ` $http.get('/users/' + id + '/cashboxes')`, it would be nice to make this `/:id/cashboxes`.  That way, it is easier for developers to find it.